docs/sdks/batches/README.md

```diff
@@ -0,0 +1,660 @@
+# Batches
+(*batches()*)
+
+## Overview
+
+A batch is a technique for creating multiple labels at once. Use the  batch object to create and purchase many shipments in two API calls. After creating the batch, retrieve the batch to verify that all shipments are valid. You can add and remove shipments after you have created the batch. When all shipments are valid you can purchase the batch and retrieve all the shipping labels.
+<SchemaDefinition schemaRef="#/components/schemas/Batch"/>
+
+# Batch Shipment
+The batch shipment object is a wrapper around a shipment object, which include shipment-specific information 
+for batch processing.
+
+Note: batch shipments can only be created on the batch endpoint, either when creating a batch object or by through 
+the `/batches/{BATCH_OBJECT_ID}/add_shipments` endpoint
+<SchemaDefinition schemaRef="#/components/schemas/BatchShipment"/>
+
+### Available Operations
+
+* [create](#create) - Create a batch
+* [get](#get) - Retrieve a batch
+* [addShipments](#addshipments) - Add shipments to a batch
+* [purchase](#purchase) - Purchase a batch
+* [removeShipments](#removeshipments) - Remove shipments from a batch
+
+## create
+
+Creates a new batch object for purchasing shipping labels for many shipments at once. Batches are created asynchronously. This means that the API response won't include your batch shipments yet. You need to retrieve the batch later to verify that all batch shipments are valid.
+
+### Example Usage
```

Comment:
I feel like these examples would be more useful in separate Java files if not a separate sample Shippo repo with a URL